in-room-controller/.eslintrc

```diff
@@ -4,6 +4,9 @@
         "eslint-config-jitsi/jsdoc",
         "eslint-config-jitsi/react"
     ],
+    "globals": {
+        "__DEV__": true
```

Comment:
How would this get set to false?

Reply:
oh I merged before seeing your reply. So this is only for the eslint. It is set by react-native to false when running in release mode

Reply:
Ah, right, true meaning it's an allowed global.